forge/src/main/java/org/openjdk/skara/forge/github/GitHubRepository.java

```diff
@@ -145,13 +145,26 @@ public PullRequest pullRequest(String id) {
 
     @Override
     public List<PullRequest> pullRequests() {
```

Comment:
I get why you want this method for symmetry, but in reality we could never actually call it in Skara if it would actually return all pull requests for a repo. The potential result set is just too large. The JDK repo has over 10k already. This is why the old `pullRequests(ZonedDateTime)` has the maxPages(1) limit. This method needs that too.

Reply:
Fixed.